pkg/cmd/service/service.go

```diff
@@ -7,14 +7,15 @@ import (
 	"log"
 	"time"
 
+	grpc_auth "github.com/grpc-ecosystem/go-grpc-middleware/auth"
 	"github.com/kelseyhightower/envconfig"
 	"github.com/openfga/openfga/pkg/encoder"
 	"github.com/openfga/openfga/pkg/logger"
 	"github.com/openfga/openfga/pkg/telemetry"
 	"github.com/openfga/openfga/server"
-	"github.com/openfga/openfga/server/authentication"
-	"github.com/openfga/openfga/server/authentication/oidc"
-	"github.com/openfga/openfga/server/authentication/presharedkey"
+	"github.com/openfga/openfga/server/authn"
```

Comment:
`authn` is ubiquitous enough to use it in place of the long-hand "authentication". We'll introduce the `authz` package when/if we introduce FGA inside of FGA via an `authz.Authorizer` interface.